From: Dmitry Shachnev <mitya57@debian.org>
Date: Sat, 8 Feb 2020 12:47:10 +0300
Subject: html_sidebars values should be lists, not single strings

The single string values were deprecated in Sphinx 1.7 and removed
in Sphinx 2.0.
---
 doc/conf.py | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/doc/conf.py b/doc/conf.py
index ac7f0cb..358414f 100644
--- a/doc/conf.py
+++ b/doc/conf.py
@@ -132,7 +132,13 @@ html_static_path = ['.static']
 
 # Custom sidebar templates, maps document names to template names.
 html_sidebars = {
-    'index': 'indexsidebar.html'
+    'index': [
+        'localtoc.html',
+        'relations.html',
+        'sourcelink.html',
+        'indexsidebar.html',
+        'searchbox.html',
+        ],
     }
 
 # Additional templates that should be rendered to pages, maps page names to
